admin/orders.php

```diff
@@ -747,6 +761,31 @@ function couponpopupWindow(url) {
                 <th class="text-center"><?php echo TABLE_HEADING_DATE_ADDED; ?></th>
                 <th class="text-center"><?php echo TABLE_HEADING_CUSTOMER_NOTIFIED; ?></th>
                 <th class="text-center"><?php echo TABLE_HEADING_STATUS; ?></th>
+<?php
+  // -----
+  // A watching observer can provide an associative array in the form:
+  //
+  // $extra_headings = array(
+  //     array(
+  //       'align' => $alignment,    // One of 'center', 'right', or 'left' (optional)
+  //       'text' => $value
+  //     ),
+  // );
+  //
+  // Observer note:  Be sure to check that the $p2/$extra_headings value is specifically (bool)false before initializing, since
```

Comment:
Based on the note of line 828, should this be `$p3/…` instead of `$p2/…` or some other value as these `p` values are not defined anywhere? Also wouldn't below a test of `!empty($extra_headings)` be more encompassing ($extra_headings gets `unset` for example)?

Reply:
Good point; need to remember to review after copy&paste!